core/io/image.cpp

```diff
@@ -2631,9 +2631,10 @@ Vector<uint8_t> Image::save_jpg_to_buffer(float p_quality) const {
 }
 
 Error Image::save_exr(const String &p_path, bool p_grayscale) const {
-	if (save_exr_func == nullptr) {
-		return ERR_UNAVAILABLE;
-	}
+	ERR_FAIL_NULL_V_MSG(
+			save_exr_func,
+			ERR_UNAVAILABLE,
+			"The TinyEXR module isn't enabled. Recompile the Godot editor with the `module_tinyexr_enabled=yes` SCons option.");
```

Comment:
Good idea but I think maybe as if tools aren't enabled there is no sense to check if `save_exr_func` is null as we already know it will be null...

Very good suggestion but I will replace it with `ERR_FAIL_V_MSG`